lib/lib.md

```diff
@@ -0,0 +1,101 @@
+# Module lib
+
+StarkNet.kt is a library allowing for easy interaction with StarkNet gateway and nodes, including
+querying starknet state, executing transactions and deploying contracts.
+
+Although written in Kotlin, StarkNet.kt has been created with compatibility with Java in mind.
+
+## Example usages in Java
```

Comment:
We can structure it a little bit better:
- Asynchronous requests
  - java
  - kotlin
- Synchronous requests
  - java
  - kotlin